lib/features/bendpoints/BendpointMovePreview.js

```diff
@@ -105,13 +105,11 @@ export default function BendpointMovePreview(bendpointMove, injector, eventBus,
         target = connection.target,
         newWaypoints = connection.waypoints.slice(),
         bendpoint = { x: event.x, y: event.y },
-        hints;
+        hints = context.hints || {};
 
     if (connectionPreview) {
-      hints = {
-        source: source,
-        target: target
-      };
+      hints.source = source;
```

Comment:
Is it intended that we monkeypatch here?

Reply:
Kind of. We need to be able to pass the `connectionStart` and `connectionEnd` hints from the DRD-specific bendpoint snapping in dmn-js so that the correct preview is rendered. This hint is not overridden here because it's always the opposite one.

Reply:
Let's not monkey patch.